subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/executer/OutputScrapingExecutionResult.java

```diff
@@ -54,7 +54,7 @@
     private final Pattern skippedTaskPattern = Pattern.compile(TASK_LOGGER_DEBUG_PATTERN + "(:\\S+?(:\\S+?)*)\\s+((SKIPPED)|(UP-TO-DATE)|(NO-SOURCE)|(FROM-CACHE))");
 
     //for example: ':hey' or ':a SKIPPED' or ':foo:bar:baz UP-TO-DATE' but not ':a FOO'
-    private final Pattern taskPattern = Pattern.compile(TASK_LOGGER_DEBUG_PATTERN + "(:\\S+?(:\\S+?)*)((\\s+SKIPPED)|(\\s+UP-TO-DATE)|(\\s+FROM-CACHE)|(\\s+NO-SOURCE)|(\\s+FAILED)|(\\s*))");
+    private final Pattern taskPattern = Pattern.compile("\\s*" + TASK_LOGGER_DEBUG_PATTERN + "(:\\S+?(:\\S+?)*)((\\s+SKIPPED)|(\\s+UP-TO-DATE)|(\\s+FROM-CACHE)|(\\s+NO-SOURCE)|(\\s+FAILED)|(\\s*))");
```

Comment:
I'm unsure if I should be changing this pattern. Basically, MSBuild adds space in front of the output of his exec tasks which break this pattern. What do you think @ghale.

Reply:
I was thinking about this, too, but I couldn't think of any known situation that this would break.  I'd say if this is clearing CI then it's fine.

One other comment I would make, though, is that I would think this should also be in `skippedTaskPattern` above as well.

Reply:
I think we should try to not change this pattern because it affects so many tests.  

As an alternative, we could also run the output into something that would trim each line before sending it through the output scraper.

Or for these kinds of tools, we could try to rely on the build operation trace more.  msbuild makes this hard since it runs Gradle multiple times, so I haven't really thought through how we'd be able to look at the operation trace for each execution.